src/core_read.cpp

```diff
@@ -109,28 +109,23 @@ static bool CheckTxScriptsSanity(const CMutableTransaction& tx)
     return true;
 }
 
-bool DecodeHexTx(CMutableTransaction& tx, const std::string& hex_tx, bool try_no_witness, bool try_witness)
+bool DecodeTx(CMutableTransaction& tx, const std::vector<unsigned char> tx_data, bool try_no_witness, bool try_witness)
```

Comment:
nit, static.

Reply:
@promag I want to be able to fuzz `DecodeTx` directly without going via `DecodeHexTx` :)

Reply:
Oh in that case dismiss my suggestion!

Reply:
@practicalswift on a second though it could be `static` for now and then you remove it and maybe also declare it in the header.

Reply:
done